### PR TITLE
Add several tsql statements

### PR DESCRIPTION
**Pull request by kevinramage**
- ISNULL function
- Include clause for index creation
- Xml declaration
- Built-in function
- Value constructor
- Stuff function
- Pivot / unpivot statement ( revision cd79677 + 6efbda6)
- Open XML statement

### CODE REVIEW

**Comment by teverett** [comment]
You seem to have some test case failures.

File: tsql/tsql.g4 [hunk 1]
```diff
@@ -221,6 +221,7 @@ create_database
 // https://msdn.microsoft.com/en-us/library/ms188783.aspx
 create_index
     : CREATE UNIQUE? clustered? INDEX id ON table_name_with_hint '(' column_name_list (ASC | DESC)? ')'
+	(INCLUDE '(' column_name_list ')') ?
```
Comment (KvanTTT) [inline tsql/tsql.g4]: Can you also replace all tabs with spaces in tsql.g4 file?

**Comment by teverett** [comment]
@kevinramage Could you please fix the merge issues?

**Comment by KvanTTT** [comment]
I'll fix it if @kevinramage won't.

**Pull request closed without merging**